package.json

```diff
@@ -37,7 +37,8 @@
     "libsodium-wrappers": "^0.5.4",
     "libsodium-wrappers-sumo": "^0.5.4",
     "tsconfig-paths-webpack-plugin": "^3.2.0",
-    "zxcvbn": "^4.4.2"
+    "zxcvbn": "^4.4.2",
+    "yarn": "^1.10.1"
```

Comment:
How come we are using Yarn now?

Reply:
this matches master
we can rip it out in the next PR

Reply:
@vishakh We can use both of npm and yarn.